frontend/src/app/main/ui/ds/foundations/typography.mdx

```diff
@@ -0,0 +1,135 @@
+import { Canvas, Meta } from "@storybook/blocks";
+import * as TypographyStories from "./typography.stories";
+
+<Meta of={TypographyStories} />
+
+## Fonts
+
+Our preferred font is "Work Sans", it is used throughout our interface.  
+In case of having the interface in Farsi or Arabic this font will be replaced by "Vazirmatn".
+
+In the code area "Roboto Mono" will be used.
+
+## Variants
+### Title variants 
+
+#### Display (default)
+
+- Used in the majority of the interface, and **it is the default variant**.
+
+  font-weight: 400;
+
+  line-height: 1.2;
+
+  font-size: 36px;
+
+#### Title large
+
+  font-weight: 400;
+
+  line-height: 1.2;
+
+  font-size: 24px;
+
+#### Title medium
+
+  font-weight: 400;
+
+  line-height: 1.2;
+
+  font-size: 20px;
+
+
+### Heading variants
+These variants are written in uppercase.
+
+#### Heading large
+
+  font-weight: 400;
+
+  line-height: 1.2;
+
+  font-size: 18px;
+
+  text-transform: uppercase;
+
+#### Heading medium
+
+  font-weight: 400;
+
+  line-height: 1.2;
+
+  font-size: 16px;
+
+  text-transform: uppercase;
+
+#### Heading small
+
+  font-weight: 500;
+
+  line-height: 1.2;
+
+  font-size: 12px;
+
+  text-transform: uppercase;
+
+### Body variants
+#### Body large
+
+  font-family: "worksans", "vazirmatn", sans-serif;
+
+  font-weight: 400;
+
+  line-height: 1.4;
+
+  font-size: 16px;
+
+#### Body medium
+  font-family: "worksans", "vazirmatn", sans-serif;
+
+  font-weight: 400;
+
+  line-height: 1.3;
+
+  font-size: 14px;
+
+#### Body small
+
+  font-family: "worksans", "vazirmatn", sans-serif;
+
+  font-weight: 400;
+
+  line-height: 1.3;
+
+  font-size: 12px;
+
+### Code variants
+### Code font
+  font-family: "Roboto Mono", monospace;
+
+  font-weight: 400;
+
+  line-height: 1.2;
+  
+  font-size: 12px;
+
+
+### Heading typography
+
+The component supports a typography prop, but it is not mandatory. 
+If the prop is not passed the heading will inherit the css from the body.
```

Comment:
If the prop is not passed for headings, an error will be yield in development env.